subprojects/gst-plugins-bad/sys/msdk/gstmsdkenc.c

```diff
@@ -171,13 +171,85 @@ gst_msdkenc_set_context (GstElement * element, GstContext * context)
   GST_ELEMENT_CLASS (parent_class)->set_context (element, context);
 }
 
+static guint
+gst_msdkenc_get_preset_target_bitrate (GstMsdkEnc * thiz)
+{
+  mfxInfoMFX *mfx = &thiz->param.mfx;
+  gdouble bitrate = 0;
+  guint low = 0, high = 0;
+  guint pair_cnt = 0;
+  gdouble (*bit_pair)[2] = NULL;
+  gdouble point;
+  guint i;
+
+  static gdouble preset_bitrate[3][5][2] = {
```

Comment:
Could you add a comment about where these preset values come from (i.e. maybe msdk/onevpl reference)?

Reply:
Added a comment. Is it acceptable?